internal/command/apps/destroy.go

```diff
@@ -35,6 +35,7 @@ from the Fly platform.
 
 	destroy.ValidArgsFunction = completion.Adapt(completion.CompleteApps)
 
+	destroy.Aliases = []string{"rm"}
```

Comment:
Why not but we probably want to standarize a bit and accept all variations:

`fly machines` can do `destroy`, `remove` and `rm`. 
`fly volumes` accepts `destroy`, `delete` and `rm`

Reply:
Happy to add more, I went conservative for a first pass but I've also tried `fly app delete` before too.